pkg/monitor/launch/launcher.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package launch
+
+import (
+	"bufio"
+	"encoding/json"
+	"os"
+	"os/exec"
+	"sync"
+
+	"github.com/cilium/cilium/api/v1/models"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const targetName = "cilium-node-monitor"
+
+var stateMutex sync.Mutex
```

Comment:
Will drop it. Added it because of some other conditional issue, don't think it's needed.